pkg/exporter/process.go

```diff
@@ -31,6 +33,7 @@ import (
 
 const startTemplateID uint16 = 255
 const defaultCheckConnInterval = 10 * time.Second
+const msgBufferLen = 4000
```

Comment:
we may end up needing a bigger record if metadata in ipfix is long, right?

Reply:
Correct. Do you have any suggested size in mind?

Reply:
May be allocate bigger size with a factor based on the length of ipfix message length?